addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -42,7 +42,9 @@
     "jasmine-spec-reporter": "^2.4.0",
     "karma": "^0.13.15",
     "karma-chrome-launcher": "^0.2.3",
+    "karma-coverage": "^1.0.0",
     "karma-jasmine": "^0.3.8",
+    "karma-remap-istanbul": "git+https://github.com/marines/karma-remap-istanbul.git#patch-1",
```

Comment:
I'm sorry but I can't allow every CLI dependencies to be reliant on a ref on someone's GitHub.